distributor/src/distributor/website.bal

```diff
@@ -1,17 +1,57 @@
-import ballerina/auth;
+import ballerina/file;
 import ballerina/http;
 import ballerina/log;
 import ballerina/mime;
 import ballerina/time;
 import ballerina/xmlutils;
-import ballerina/file;
-import ballerina/websub;
 
 const LEVEL_PD = "POLLING-DIVISION";
 const LEVEL_ED = "ELECTORAL-DISTRICT";
 const LEVEL_NI = "NATIONAL-INCREMENTAL";
 const LEVEL_NF = "NATIONAL-FINAL";
 
+const WANT_IMAGE = "image";
+const WANT_AWAIT_RESULTS = "await";
+
+map<http:WebSocketCaller> jsonConnections = {};
+map<http:WebSocketCaller> imageConnections = {};
+map<http:WebSocketCaller> awaitConnections = {};
```

Comment:
Do we need 3 maps here? Is it to avoid potential concurrent issues? Isn't all three maps contain same set of connections?

Reply:
We only send the image data and await results notification if the recipient explicitly requests them. So while the `jsonConnections` map will have all the recipients, not all of them may be in `imageConnections` or `awaitConnections`.